components/src/core/components/TableSidebar/TableSidebar.vue

```diff
@@ -2,25 +2,30 @@
   <div class="oxd-table-left-panel" :style="customStyles">
     <div v-if="headerVisible" class="oxd-table-left-panel--header">
       <slot name="header">
-        <oxd-button
-          class="table-header-action-btn"
-          :class="!isLeftPanelOpen ? 'no-label' : 'w-100'"
-          :tooltip="!isLeftPanelOpen ? buttonData.label : null"
-          flow="right"
-          :label="buttonData.label"
-          :iconName="buttonData.iconName"
-          :iconSize="buttonData.iconSize"
-          :size="buttonData.size"
-          :style="buttonData.style"
-          :displayType="buttonData.displayType"
-          @click="onHeaderBtnClick"
+        <div
```

Comment:
Will there by any css padding isues due to this?

Reply:
I fixed possible ones